core/src/nucleus/ribosome.rs

```diff
@@ -95,6 +94,7 @@ pub fn call(wasm: Vec<u8>, function_name: &str) -> Result<Runtime, InterpreterEr
 
 #[cfg(test)]
 mod tests {
+    extern crate wabt;
```

Comment:
`extern crate` statement should only be at the very top of the root file. This is an established standard in the Rust community that make crate lookup for developers really easy

Reply:
@thedavidmeister Yeah, this makes this even more confusing. @lucksus If the crate is not used anywhere except the test environment, the compiler will take care of that. The compiler does LTO.

Reply:
No, it's not about the compiler. When I switch the toolchain to armv7-linux-androideabi, wabt won't compile it's C bindings on install. That's why I had to do this.

Also, did you see that I've moved the crate definition up and out of this module? This convo has become superfluous ;)